readthedocs/templates/core/project_detail_right.html

```diff
@@ -67,8 +67,13 @@ <h3>{% trans "Home Page" %}</h3>
 
 {% block badge %}
     <h3>{% trans "Badge" %}</h3>
+      <img src="{{ badge_url }}" />
       <a href="{{ site_url }}" style="text-decoration: none;" onclick="$('#badge_markup').toggle(); return false;">
-        <img src="{{ badge_url }}" />
+        <span title="Get this badge">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50" width="20px" height="20px">
+            <path style="line-height:normal;text-indent:0;text-align:start;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000;text-transform:none;block-progression:tb;isolation:auto;mix-blend-mode:normal" d="M 25 2 C 12.309295 2 2 12.309295 2 25 C 2 37.690705 12.309295 48 25 48 C 37.690705 48 48 37.690705 48 25 C 48 12.309295 37.690705 2 25 2 z M 25 4 C 36.609824 4 46 13.390176 46 25 C 46 36.609824 36.609824 46 25 46 C 13.390176 46 4 36.609824 4 25 C 4 13.390176 13.390176 4 25 4 z M 25 11 A 3 3 0 0 0 22 14 A 3 3 0 0 0 25 17 A 3 3 0 0 0 28 14 A 3 3 0 0 0 25 11 z M 21 21 L 21 23 L 22 23 L 23 23 L 23 36 L 22 36 L 21 36 L 21 38 L 22 38 L 23 38 L 27 38 L 28 38 L 29 38 L 29 36 L 28 36 L 27 36 L 27 21 L 26 21 L 22 21 L 21 21 z" font-weight="400" font-family="sans-serif" white-space="normal" overflow="visible"/>
```

Comment:
I think we can use fontawesome here

Reply:
Is RTD using font awesome?
I tried but the icon doesn't appear.

Reply:
Yes we are, can't remember the version, tho

Reply:
I did it with fontawesome. Now it looks like this:

![screenshot from 2019-01-29 12-43-30](https://user-images.githubusercontent.com/29149191/51891066-ffe05e80-23c3-11e9-90f4-6a02e896ae05.png)